sleuth_crawler/scraper/scraper/spiders/ubc_homepage_spider.py

```diff
@@ -0,0 +1,38 @@
+import scrapy
+import re
+from scrapy.contrib.spiders import Rule, CrawlSpider
+from scrapy.linkextractors import LinkExtractor
+from scraper.items import GenericPage
+
+class UbcBroadCrawler(CrawlSpider):
+    """
+    Spider that crawls generically starting at UBC's homepage
+    """
+    name = "ubc_broad"
+
+    start_urls = ["http://www.ubc.ca"]
+    allowed = ['ubc', 'universityofbc']
+    rules = (
+        Rule(
+            LinkExtractor(
+                allow=('ubc', 'universityofbc', ), 
+                deny=('youtube', 'twitter', 'facebook')
+            ),
+            callback='parse_item'
+        ),
+    )
+    custom_settings = {
+        'ITEM_PIPELINES': {
+            'scraper.pipelines.SolrPipeline': 400,
+            'scraper.pipelines.JsonLogPipeline': 300,
+        }
+    }
+
+    def parse_item(self, response):
+        """
+        Scrape page
+        """
+        output = GenericPage()
+        output['url'] = response.url
+        output['page_data'] = re.sub(r'<[^>]*?>', '', str(response.body))
```

Comment:
This is good for now, but later we should get fancy about stripping HTML/JS and additional whitespace, so call that that's left is a simple text document.